Update Get-CsOnlineUser.md (#10666)
MicrosoftDocs/office-docs-powershell: skype/skype-ps/skype/Get-CsOnlineUser.md

The Filter parameter uses the same filtering syntax as the Where-Object cmdlet. For example, the following filter returns only users who have been enabled for Enterprise Voice: `-Filter 'EnterpriseVoiceEnabled -eq $True'` or ``-Filter "EnterpriseVoiceEnabled -eq `$True"``.

> **Updates in Teams PowerShell Module version 5.0.0 and above**

Reviewer: ```suggestion
**Updates in Teams PowerShell Module version 5.0.0 and later**
```

> The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell versions >=5.0.0 and will be applicable to all Teams PowerShell Module versions by 15th April 2023 (note that these changes are only rolled out in commercial environments at present):

Reviewer 2: ```suggestion
The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell version 5.0.0 or later (note that these changes are only rolled out in commercial environments at present):
```

> **Updates in Teams PowerShell Module version 3.0.0 and above**

Reviewer: ```suggestion
**Updates in Teams PowerShell Module version 3.0.0 and later**
```

Accept wildcard characters: False
```

> ### -Sort/-OrderBy

Reviewer 3: We can't use more than one parameter name here, please split in two and also add the yaml section to conform with cmdlet's article format

Author: @get-itips I've modified it. Thanks!

Sorting is now enabled in Teams PowerShell Module 5.1.0 and above by using the "-Sort" or "-OrderBy" parameters. For Example:

- Get-CsOnlineUser -Filter {LineURI -like *123*} -OrderBy "DisplayName asc"
- Get-CsOnlineUser -Filter {DisplayName -like '*abc'} -OrderBy {DisplayName desc}

### -UnassignedUser

> This parameter has been deprecated from the Teams PowerShell Modules version 3.0 or later due to limited usage.

Reviewer: ```suggestion
This parameter has been deprecated from the Teams PowerShell Modules version 3.0 and later due to limited usage.
```

**Updates in Teams PowerShell Module version 5.0.0**

> The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell version >=5.0.0 and will be applicable to all Teams PowerShell Module versions by 15th April 2023  (note that these changes are only rolled out in commercial environments at present):

Reviewer 2: ```suggestion
The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell version 5.0.0 or later (note that these changes are only rolled out in commercial environments at present):
```
